tests/plugins/test_tag_and_push.py

```diff
@@ -121,7 +121,12 @@ def test_tag_and_push_plugin(tmpdir, monkeypatch, image_name, logs, should_raise
 
     (flexmock(atomic_reactor.util)
         .should_receive('get_manifest_digests')
-        .and_return({'v1': DIGEST_V1, 'v2': DIGEST_V2})
+        .and_return(ManifestDigest(v1=DIGEST_V1, v2=DIGEST_V2))
+    )
+
+    (flexmock(atomic_reactor.util)
+        .should_receive('get_config_from_registry')
+        .and_return({'id': '123', 'container_config': {}, 'config': {}, 'docker_version': '1.9.1'})
```

Comment:
In addition this would mean we'd get coverage here:
https://coveralls.io/builds/8279004/source?filename=atomic_reactor%2Fplugins%2Fpost_tag_and_push.py#L89